BondageClub/Scripts/Inventory.js

```diff
@@ -528,6 +533,13 @@ function InventoryRemove(C, AssetGroup, Refresh) {
 		if (C.Appearance[E].Asset.Group.Name == AssetGroup) {
 			C.Appearance.splice(E, 1);
 			if (Refresh || Refresh == null) CharacterRefresh(C);
+
+			if (Player.GraphicsSettings && Player.GraphicsSettings.DoBlindFlash) {
+				if (Player.GetBlindLevel() < lastblindlevel && Player.GetBlindLevel() == 0) {
```

Comment:
```suggestion
				if (lastblindlevel > 0 && Player.GetBlindLevel() === 0) {
```
Just a minor one here, but `Player.GetBlindLevel` actually encapsulates a fair bit of logic, so it's probably worth just calling it once here.